libs/langchain-community/src/llms/tests/ibm.test.ts

```diff
@@ -156,17 +153,6 @@ describe("LLM unit tests", () => {
             ...fakeAuthProp,
           })
       ).toThrowError();
-      // @ts-expect-error Intentionally passing not enough parameters
-      const testPropsServiceUrl: WatsonxInputLLM = {
-        serviceUrl: process.env.WATSONX_AI_SERVICE_URL as string,
-      };
-      expect(
-        () =>
-          new WatsonxLLM({
-            ...testPropsServiceUrl,
-            ...fakeAuthProp,
-          })
-      ).toThrowError();
```

Comment:
Assuming it's possible to check for the lightweight engine, you could keep this logic here, but pass whatever is necessary to detect that use